charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,9 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.10.6
+version: 12.3.1
+appVersion: 0.43.2
+version: 12.10.7
```

Comment:
hi, please remove the duplicate keys. Ref lint errors 

```charts/kube-prometheus-stack/Chart.yaml
  23:1      error    duplication of key "version" in mapping  (key-duplicates)
  24:1      error    duplication of key "appVersion" in mapping  (key-duplicates)```

Reply:
This is fixed